soc/arm/renesas_rcar/gen3/Kconfig.series

```diff
@@ -10,5 +10,6 @@ config SOC_SERIES_RCAR_GEN3
 	select CPU_HAS_DCLS
 	select SOC_FAMILY_RCAR
 	select CLOCK_CONTROL_RCAR_CPG_MSSR if CLOCK_CONTROL
+	select PINCTRL
```

Comment:
I'd suggest instead this in `Kconfig.defconfig`

```c
config PINCTRL
    default y
```

(doesn't make it a strong requirement)